### DISCUSSION TITLE
WikiJS store as markdown only

### DISCUSSION THREAD

**Discussion by Atomique** [requarks/wiki#5630] [Help / Questions] [answered]
Hello, 

I added a gitlab repo as storage. I would like to store anything as markdown inside my repo to be able to view it in gitlab also. I have created a page and have seen, that WikiJS transformed it to html (with the rendering engine I assume). 
I wondwr if it is possible to just store anything as markdown and be able to use it WikiJS and Gitlab (or future applications). 

Thanks in advance!

**Comment by NGPixel** [comment, marked as answer]
Use the Markdown editor when creating pages and it will be saved as markdown files. The visual editor will always save pages as html.

**Reply by Atomique** [reply]
Okay that was easy, thanks a lot. That solved my problem!
